sample-functions/ApiKeyProtected/README.md

```diff
@@ -1,6 +1,6 @@
 ### Api-Key-Protected sample
 
-To use this sample provide an env variable for the container/service in `secret_api_key`.
+See the [secure secret management guide](../guide/secure_secret_management.md) for instructions on how to use this function.
```

Comment:
Are there plans to create this file or does it exist somewhere else?

Reply:
We should fix the link @LucasRoesler